ods_ci/tests/Resources/Common.robot

```diff
@@ -200,17 +200,31 @@ Get CodeFlare Version
 #robocop: disable: line-too-long
 Wait Until Csv Is Ready
```

Comment:
maybe both `Wait Until Csv Is Ready` and `Search Install Plan` should be moved to `ods_ci/tests/Resources/OCP.resource`